src/storage/src/hummock/sstable_store.rs

```diff
@@ -375,7 +375,7 @@ impl SstableStore {
 
     pub async fn list_ssts_from_object_store(&self) -> HummockResult<Vec<ObjectMetadata>> {
         self.store
-            .list(&self.path)
```

Comment:
Shall we need to add some formatting checks to the `path`? Adding "/" directly is not necessarily the right thing to do. We can check it at path initialization and use the same rule here to perform assert here

Reply:
We should validate path. Will leave a TODO.

Reply:
This added "/" always matches exactly the same "/" added in [get_sst_data_path](https://github.com/risingwavelabs/risingwave/blob/f309b246571a9ec68201d696fbcd5cfd4181519f/src/storage/src/hummock/sstable_store.rs#L288). It aims to filter out path with same prefix.